tests/sentry/api/endpoints/test_project_details.py

```diff
@@ -214,7 +214,6 @@ def test_platform(self):
 
     def test_options(self):
         options = {
-            'sentry:origins': 'foo\nbar',
```

Comment:
was this accidental?

Reply:
No, I asked @LSSangha what to do, since he was on the git blame for this.

Reply:
Was it causing flake8 to complain? Just curious why the change since inherently it would test something (though might not matter)

Reply:
Yes, since `'sentry:origins'` is a duplicated key. It was originally `'sentry:origins': 'foo\nbar'`, then he added `'sentry:origins': '*'`. I asked him and he said it was best to keep his change, which was being tested on anyways since the key value definition appears later, so it overrides the old one.